packages/select/src/react/index.tsx

```diff
@@ -43,39 +43,40 @@ interface SelectProps extends UseListboxProps {
   renderOption?: React.FC
 }
 
-const Select = forwardRefWithAsAndStatics<SelectProps, 'button', SelectStatics>(
-  (props, ref) => {
-    const {
-      options = [],
-      position = 'belowLeft',
-      renderOption = defaultRenderOption,
-      children,
-      ...rest
-    } = props
-    const { buttonProps, selectedProps, menuProps, isOpen } = useListbox(
-      rest,
-      ref
-    )
+const Select = forwardRefWithStatics<
```

Comment:
Double-checking: Do we not need to support the `as` prop?

Reply:
Not on Select. Only the Menu.Item needs that.